input/cql/IntravesicalBacillusCalmetteGuerinForBladderCancerFHIR.cql

```diff
@@ -135,6 +145,9 @@ define "Has Qualifying Encounter":
         and ValidEncounter.status = 'finished'
   )
 
+/*
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
As written this should also be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

Reply:
Updated comment to be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis.